sys-fs/mergerfs/mergerfs-2.24.2.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs autotools
+
+DESCRIPTION="A featureful FUSE union filesystem across multiple drives"
+HOMEPAGE="https://github.com/trapexit/mergerfs"
+# tarball under download/ has prebuilt ChangeLog and AUTHORS,
```

Comment:
Dedicated release tarballs are always preferable over snapshots, so there's no need to explain it in a comment.

Reply:
Well, release tags can be verified against the author's GPG signature, while (in this particular case) release tarballs can not.
Also, in this case, the release tarball contains pre-compiled binaries, which is not always desired.
That's why I tried to explain, what's particularly useful in the release tarball.

I can surely remove the comment per your suggestion.

Reply:
But autogenerated github tarballs don't contain enough data to verify a signature either, and we can't use git directly. Prebuilt files aren't much of an issue as long as they end up being recompiled.

Reply:
In fully automated way -- no.
But manually, I could have cloned the repo, verified the tag signature, generated a tarball from the tag, and compared it to the one obtained from github; and then set the right checksum in `Manifest` file.

Anyway, I'll remove the comment.